config/checkstyle_input_suppressions.xml

```diff
@@ -1582,8 +1582,6 @@
             files="sizes[\\/]methodlength[\\/]InputMethodLengthComments.java"/>
   <suppress id="ConfigCommentOnTopInputs"
             files="sizes[\\/]methodlength[\\/]InputMethodLengthModifier.java"/>
```

Comment:
please remove all suppressions for `methodlength` inputs

Reply:
On it.

Reply:
Done.